cdc/generation.cc

```diff
@@ -330,7 +330,7 @@ topology_description limit_number_of_streams_if_needed(topology_description&& de
     return topology_description(std::move(entries));
 }
 
-future<cdc::generation_id> make_new_cdc_generation(
+future<cdc::generation_id_v1> make_new_cdc_generation(
```

Comment:
Comment to commit msg:
```
 and how to persist new identifiers in local tables.
```
Is this really the case?

Reply:
Yes, `update_cdc_generation_id` and `get_cdc_generation_id`, which update and retrieve the ID from the local table, has been modified in this commit to handle both types of identifiers.

So nodes can handle the new IDs after this commit but they don't create these IDs yet (in this commit).